internal/makefile/makefile.go

```diff
@@ -136,6 +138,34 @@ endif
 		prepareStaticRecipe = append(prepareStaticRecipe, "install-shellcheck")
 	}
 
+	if cfg.Typos.IsEnabled() {
+		prepare.addRule(rule{
+			description: "Install typos required by run-typos/static-check",
+			phony:       true,
+			target:      "install-typos",
+			recipe: []string{
+				// see https://github.com/crate-ci/typos/blob/master/action/entrypoint.sh
+				`@set -eou pipefail; ` +
+					` if ! hash typos 2>/dev/null; then` +
+					` printf "\e[1;36m>> Installing typos...\e[0m\n";` +
+					` TYPOS_ARCH=$$(uname -m);` +
+					// relevant for MacOS
+					` if [[ "$$TYPOS_ARCH" == "arm64" ]]; then TYPOS_ARCH=aarch64; fi;` +
+					` if command -v gh >/dev/null; then TYPOS_GET_RELEASE_JSON="gh api /repos/crate-ci/typos/releases"; else TYPOS_GET_RELEASE_JSON="curl -s https://api.github.com/repos/crate-ci/typos/releases"; fi;` +
+					` TYPOS_VERSION=$$($$TYPOS_GET_RELEASE_JSON | jq -r '.[0].name' );` +
```

Comment:
Is there an advantage in going through `gh` if available? It feels unlikely to me that we would have `gh` installed, but not `curl`.